tests/cloudformation/checks/resource/aws/example_WorkspaceUserVolumeEncrypted/PASS.yaml

```diff
@@ -0,0 +1,9 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Resources: 
+  Pass:
+    Type: AWS::WorkSpaces::Workspace
+    Properties: 
+      UserName: test
+      BundleId: wsb-abc123456
+      DirectoryId: d-abc123456
+      UserVolumeEncryptionEnabled: True
```

Comment:
@njgibbon Shouldn't it be yaml boolean, i.e. `true`?

Reply:
hey @nimrodkor  - Good point, good catch! I'll update when I have a bit of time spare.

Reply:
@nimrodkor  Update now :-)